iohub/cli/cli.py

```diff
@@ -87,3 +89,60 @@ def convert(input, output, grid_layout, chunks):
         chunks=chunks,
     )
     converter()
+
+
+@click.command()
+@click.help_option("-h", "--help")
+@click.argument(
+    "csvfile",
+    type=click.File("r"),
+)
+@click.argument(
+    "zarrfile",
+    type=click.Path(
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
```

Comment:
I'm in favor of options instead of arguments (largely to match our conventions in shrimpy).

Maybe a signature like: `iohub rename-wells -i input.zarr -c names.csv`